src/graphiql-explain/utils/index.js

```diff
@@ -100,6 +100,12 @@ export function getColorByLimit(maxLimit) {
   )
 }
 
+export function getSymbolByLimit(maxLimit) {
+  return (
+    reverseThresholds.find(({ limit }) => maxLimit >= limit)?.symbol || null
+  )
+}
```

Comment:
is there a particular reason to return null?
if not might wanna return undefined
```suggestion
export function getSymbolByLimit(maxLimit) {
  const threshold =  reverseThresholds.find(({ limit }) => maxLimit >= limit)
  return threshold?.symbol
}
```

Reply:
Yep, that is correct. Just the old habit of not returning undefined when something would be rendered by react as it would throw an error before.